tests/stream/test_stream_smoke/0028_replay_stream.json

```diff
@@ -0,0 +1,618 @@
+{
+    "test_suite_name": "replay_stream",
+    "tag": "smoke",
+    "test_suite_config": {
+        "setup": {
+            "statements": []
+        },
+        "tests_2_run": {
+            "ids_2_run": [
+                "all"
+            ],
+            "tags_2_run": [],
+            "tags_2_skip": {
+                "default": [
+                    "todo",
+                    "to_support",
+                    "change",
+                    "bug",
+                    "sample",
+                    "native_not_support"
+                ],
+                "cluster": [
+                    "view",
+                    "cluster_table_bug"
+                ]
+            }
+        }
+    },
+    "comments": "Tests covering replay stream",
```

Comment:
Tests mainly points:
1) tail query with different replay speed.
2) tail query from the data with different ingest interval